databroker/tests/test_pipelines.py

```diff
@@ -78,12 +82,150 @@ def multiply_by_two(arr):
     input_stream = db.restream(input_hdr, fill=True)
     output_stream = multiply_by_two(input_stream)
     for name, doc in output_stream:
+        # pprint(name)
+        # pprint(doc)
+        if name == 'start':
+            assert doc['parents'] == [input_hdr['start']['uid']]
+            output_uid = doc['uid']
+        if name == 'stop':
+            assert doc['exit_status'] == 'success'
+        if name == 'event':
+            assert_array_equal(doc['data']['image'], 2 * IMG)
+            assert 'filled' in doc.keys()
+    output_hdr = db[output_uid]
+    for ev1, ev2 in zip(db.get_events(input_hdr, fill=True),
+                        db.get_events(output_hdr, fill=True)):
+        pprint(ev1)
+        pprint(ev2)
+        AAA
```

Comment:
It was designed to make it crash, hence the work in progress. However the tests did not crash :(